src/main/java/uk/gov/hmcts/reform/civil/launchdarkly/FeatureToggleService.java

```diff
@@ -95,7 +95,7 @@ public boolean isAccessProfilesEnabled() {
     }
 
     public boolean isPbaV3Enabled() {
-        return internalClient.boolVariation("pba-version-3-ways-to-pay", createLDUser().build(), false);
+        return internalClient.boolVariation("pba-version-3-ways-to-pay", createLDUser().build(), true);
```

Comment:
Must be changed back before merge to master

Reply:
Good to know. I'll mark the code review as `Request changes` to gate any merge.